ios/FluentUI/Command Bar/CommandBarButton.swift

```diff
@@ -0,0 +1,99 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+class CommandBarButton: UIButton {
+    let item: CommandBarItem
+
+    override var isHighlighted: Bool {
+        didSet {
+            updateStyle()
+        }
+    }
+
+    override var isSelected: Bool {
+        didSet {
+            updateStyle()
+        }
+    }
+
+    override var isEnabled: Bool {
+        didSet {
+            updateStyle()
+        }
+    }
+
+    init(item: CommandBarItem, isFixed: Bool = false) {
+        self.item = item
```

Comment:
Currently, to change the state of the buttons programmatically, the developer changes the properties of `CommandBarItem`, then call `CommandBar.updateButtonsState()`. It would be not much different from implementing a delegate pattern with batch update support.

Reply:
+1 on removing the call to updateButtonsState

Reply:
This likely could be a future enhancement tied to changes for more dynamic updates to the command items, can we track it with that future work @rdeassis ?